src/index.js

```diff
@@ -52,17 +52,24 @@ client.on("ready", () => {
 
 client.on("message", (message) => {
 	if (!message.guild) return; // Ignores DMs
-	const text = message.content.toLowerCase();
-	regularQualifyCheck(message);
-
-	// Sends typing game explanation to exercise channel
-	gameExplanation(message);
 
 	if (message.author.bot) {
 		if (message.type === "PINS_ADD") message.delete();
 		return; // Ignores messages from bots
 	}
 	if (message.type === "PINS_ADD") return; // Ignores PIN messages
+
+	const text = message.content.toLowerCase();
+
+	// Filters Explicit Words
+	if (explicitWordFilter(message)) {
+		return;
+	}
+
+	regularQualifyCheck(message);
```

Comment:
This could have stayed at the top but I thought peoples' sweary messages shouldn't qualify them for the regular role 😤